libs/shared/util-date-fns/src/lib/shared-util-date-fns.spec.ts

```diff
@@ -4,4 +4,12 @@ describe('sharedUtilDateFns', () => {
   it('should convert date', () => {
     expect(normalizeDate("2015-10-04T00:00:00.000Z")).toEqual('2015-10-04');
   });
+
+  it('should not accept malformed date', () => {
```

Comment:
What about using [it.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) syntax to avoid duplication ?

Reply:
Mmh, didn't know this existed